src/composables/useScrollMonitor.ts

```diff
@@ -0,0 +1,18 @@
+import scrollMonitor from 'scrollmonitor';
```

Comment:
How this file is a composable? It doesnt seem to use any ref or computed val, it sound more like a normal function, this would fit better in helper / utils.

Reply:
Yes, good point